station/waterfall.py

```diff
@@ -0,0 +1,156 @@
+# This file is heavily based on waterfall.py from satnogs-client.
+
+import matplotlib
+import numpy as np
+import sys
+
+matplotlib.use('Agg')
+
+import matplotlib.pyplot as plt
+
+OFFSET_IN_STDS = -2.0
+SCALE_IN_STDS = 8.0
+
+class Waterfall():
```

Comment:
You can generate waterfall using ```sox``` by call:
``` sox <wav file> -n spectrogram -o <output file>```

Reply:
That's a different kind of a spectrogram, a more narrow. It shows the bandwidth that was decoded as WAV file, which is limited to whatever the sampling frequency of the WAV is. I think the max is 48k. Imagine the input band being wider, such as Meteor, which IIRC has a bandwidth of 120k. The remaining (120-48) would not be shown.

These are early experiments with histograms so I may be wrong, but the way the GR generates the waterfall has benefits. First, it's less processed, so you see the actual raw performance of the radio (antenna+SDR) system. This will show things like frequency drift, doppler problems, noise, other transmissions in nearby frequencies if the band selected is too wide, etc.

Also, this recipe does not use sox. OTOH, I think we can surely extend the classic pipeline with a histogram generated by sox. Would that work? As I said above, I'd like to keep two alternative pipelines.

Reply:
Created #52 to address this.